tests/base/test_persistor.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import mock
+
+from rasa_nlu import persistor
+
+
+class Object(object):
+    pass
+
+
+def test_if_persistor_class_has_list_projects_method():
+    with pytest.raises(NotImplementedError):
+        persistor.Persistor().list_projects()
+
+
+def test_list_projects_method_in_AWSPersistor():
+    def mocked_init(self, *args, **kwargs):
+        self._project_and_model_from_filename = lambda x: {'project_key': ('project', 'model')}[x]
+        self.bucket = Object()
```

Comment:
can't we just use `object()` and avoid the `Object` class?

Reply:
Well, `Object` can not be simple replaced by `object`, because it needs to assign an attribute to this variable which object of `object` do not support attribute assignment. If you think use `Object` class not good, It also can be implement by a `namedtuple` object. I am not an export of unit-test or python so if you have better idea I will accept it.